app/assets/stylesheets/application.scss

```diff
@@ -2,6 +2,7 @@
 @import 'uswds_overrides/all';
 @import 'c3.min';
 @import 'source/accessible-mega-menu';
+@import 'selectize.bootstrap3';
```

Comment:
nope this works as-is. comes from https://github.com/selectize/selectize.js/tree/master/dist/css